doc/reference/usermode/index.rst

```diff
@@ -41,15 +41,41 @@ For threads running in a non-privileged CPU state (hereafter referred to as
   memory that has an incompatible policy, such as attempting to write to a
   read-only area.
 
-  - Threads are automatically granted access to their own stack memory
-    region, and all other stacks are inaccessible.
+  - A user thread's access to memory is controlled by its memory domain
+    membership. A thread will always be a member of exactly one memory domain.
```

Comment:
maybe I missed something, but shouldn't be "exactly at least one memory domain" ?

Reply:
Oh no, it's never been possible for a thread to be a member of more than one memory domain. I don't know what even that would mean, semantically.

Reply:
indeed !